autoload/wiki/page.vim

```diff
@@ -125,6 +125,19 @@ function! wiki#page#rename_ask() abort "{{{1
   redraw!
   echo 'Enter new name (without extension):'
   let l:name = input('> ')
+  
+  " Check if directory exists
+  let l:newpath = printf('%s/%s', expand('%:h'), l:name)
```

Comment:
You should use `expand('%:p:h')` to ensure you are using an absolute path. I would also move the `simplify` from line 131 to 130. I would also consider to combine lines 130 and 131, since `l:newpath` is not used any further. E.g.:

```vim
let l:target_dir = fnamemodify(
      \ simplify(printf('%s/%s', expand('%:h'), l:name)), ':h')
```

Finally, note that the `expand()` on line 131 is not necessary.